service/msteams/ms_teams_test.go

```diff
@@ -0,0 +1,88 @@
+package msteams
+
+import (
+	"context"
+	"testing"
+
+	teams "github.com/atc0005/go-teams-notify/v2"
+	"github.com/google/go-cmp/cmp"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNew(t *testing.T) {
+	t.Parallel()
+
+	assert := require.New(t)
+
+	service := New()
+	assert.NotNil(service)
+}
+
+func TestMSTeams_AddReceivers(t *testing.T) {
```

Comment:
I like that the prefix points to the service name. If we're going to keep this convention in the test files, I will open another PR today, renaming the rest of the test functions accordingly, to be consistent.

Reply:
If you are interested and have the time to change that, that would of course be great!

Reply:
Will do!